src/ddmd/mars.d

```diff
@@ -1164,7 +1163,7 @@ Language changes listed by -transition=id:
     Module._init();
     Target._init();
     Expression._init();
-    objc_tryMain_init();
+    objc.initialize();
```

Comment:
Ok, so to fix this name and remove the renamed import, should I move the `initialize` function to a static method inside the `Objc` interface and name it `_init`?